src/components/UI/release_component_label.js

```diff
@@ -0,0 +1,113 @@
+import { css } from '@emotion/core';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+const radius = '5px';
```

Comment:
could this come from the theme instead? https://github.com/giantswarm/happa/blob/master/src/components/UI/theme.js

Reply:
As there is no 5px radius used by any other component, I think no. Possibly in the future.